config/sidekiq.yml

```diff
@@ -1,4 +1,6 @@
 :queues:
+  - [ daily_alert_email, 1 ]
```

Comment:
I don't think this queue gets used anymore, as the daily alert email will go to the `mailers` queue by default

Reply:
I think it's still in there for use https://github.com/dxw/teacher-vacancy-service/blob/feature/job-seeker-alerts/app/jobs/send_daily_alert_email_job.rb#L2 and nothing is using the `mailers` queue. 

Is there a reason that we're using a more generically named queue? I'm wondering how this will extend if we wanted to send another _less important_ email to users, perhaps when a job listing has expired with some stats and a call for feedback as has been discussed already.

Reply:
Ah, right you are. The reason we're using the `mailers` queue is because that's what `deliver_later` uses by default. I think it's easy to configure though.

Reply:
On testing at the moment there are queues for both names:

```
daily_alert_email
mailers
```

I'm going to see if I can set this with an explicit queue name of priority 3.

Reply:
Oh I think I get it.. queue `daily_alert_email` is for the jobs that enqueue the emails to be sent via `mailers`. Will see if I can find clearer names to avoid future confusion.